spec/texter_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "texter"
+
+describe Texter do
+  it { expect(subject).to respond_to(:send_text).with(1).argument }
+  it { expect(subject).to respond_to(:send_text).with(2).argument }
+  # I couldn't work out how to test this class further. Are we meant to mock twilio?
```

Comment:
I mocked it to test that #create was called on #message, which would then send the text.